tests/nightly/test_large_array.py

```diff
@@ -185,7 +186,19 @@ def test_pick():
     b = mx.nd.ones(shape=(256*35,))
     res = mx.nd.pick(a,b)
     assert res.shape == b.shape
-
+    
+def test_depthtospace():
```

Comment:
what are you testing here?

Reply:
Have added test cases to compare with numpy implementation

Reply:
nit: rename to `test_depth_to_space`